enterprise/frontend/src/embedding-sdk/components/private/AppInitializeController.tsx

```diff
@@ -1,13 +1,16 @@
-import type { ReactNode } from "react";
-import { t } from "ttag";
+import { type ReactNode, useMemo } from "react";
```

Comment:
heads up that I'm removing this component in the style leak PR, so we'll have to figure out what to do

Reply:
@npretto Sounds good, I'll update the PR once that happens. We can just create a dedicated component for this, like `SdkLicenseCheckWrapper` that focuses on license check and indicator.

Reply:
@npretto fixed in https://github.com/metabase/metabase/pull/47821/commits/4b8eade1d50502339a379898469c8e9f27f40250, I moved it out to be its own component.